CONTRIBUTING.md

```diff
@@ -2,34 +2,49 @@
 
-## Follow, Star & Subscribe
-[Follow us](https://twitter.com/pipedream) on Twitter, [star](https://github.com/PipedreamHQ/pipedream) our Github repo and [subscribe](https://www.youtube.com/pipedreamhq) to our YouTube channel.
+## Engage in our Community
 
-## Recommend Us!
-There’s no better way of supporting Pipedream than recommending us to your friends and colleagues.
+### Ask & Answer Questions
 
-## Ask & Answer Questions
 Do you have a question about Pipedream or how to solve a problem? Please ask us in our [community forum](https://pipedream.com/community)! Chances are that someone else has a similar question, so by helping you, we can help others encountering the same questions and challenges. Or pay it forward and help answer questions from other users.
 
-## Share Your Workflows
+### Share Your Workflows
+
 If you have a workflow that others may find interesting -- whether it’s just a fun demo or solving a real problem -- make it public and share it in our [community](https://pipedream.com/community/c/show-tell/8) or on Twitter. We may feature it to help new users get started and give others new ideas to try out.
 
+## Support Pipedream
+
+### Follow, Star & Subscribe
+
+[Follow us](https://twitter.com/pipedream) on Twitter, [star](https://github.com/PipedreamHQ/pipedream) our Github repo and [subscribe](https://www.youtube.com/pipedreamhq) to our YouTube channel.
+
+## Recommend Us!
+
+There’s no better way of supporting Pipedream than recommending us to your friends and colleagues.
+
 ## Create Content
+
 Write a blog post or create a video about how you use Pipedream. Also send it to us so we can list it on our site to send you some traffic! We also love to join live streams — just [contact us](https://pipedream.com/support/)!
 
-## Report Bugs
+### Submit Updates to Docs
+
+See a gap or mistake in our [docs](https://github.com/PipedreamHQ/pipedream/tree/master/docs)? Create a PR to submit an update.
+
+[You can setup](https://github.com/PipedreamHQ/pipedream/tree/master/docs) your own documentation development environment locally too.
+
+### Report Bugs
 
 Found an issue? [Report it](https://pipedream.com/community/c/bugs/9)
 
-## Submit & Vote On Feature Requests
+### Submit & Vote On Feature Requests
 
 Help us build an amazing product! Submit [feature requests](https://pipedream.com/community/c/feature-requests/7) in our community forum, or [browse existing requests](https://github.com/PipedreamHQ/pipedream/issues) on Github and vote for the features want us to add!
 
-## Request Apps
-Need an app we don’t support? [Request it](https://github.com/PipedreamHQ/pipedream/issues/new?assignees=&labels=app%2C+enhancement&template=app---service-integration.md&title=[APP])
+### Request Apps
 
-## Submit Updates to Docs
-See a gap or mistake in our [docs](https://github.com/PipedreamHQ/pipedream/tree/master/docs)? Create a PR to submit an update.
+Need an app or integration we don’t currently support?
```

Comment:
I would recommend linking [here](https://pipedream.com/docs/components/guidelines/#process) since it describes the process of requesting the app, building components, etc.